src/main.cpp

```diff
@@ -195,8 +195,8 @@ std::unique_ptr<bcache::program_wrapper_t> find_suitable_wrapper(
                 << (bcache::config::disable() ? "true" : "false") << "\n";
       std::cout << "  BUILDCACHE_READ_ONLY:              "
                 << (bcache::config::read_only() ? "true" : "false") << "\n";
-      std::cout << "  BUILDCACHE_ACCURACY:               "
-                << to_string(bcache::config::accuracy()) << "\n";
+      std::cout << "  BUILDCACHE_ACCURACY:               " << to_string(bcache::config::accuracy())
+                << "\n";
     }
   } catch (const std::exception& e) {
```

Comment:
We should also print out the `BUILDCACHE_COMPILER` value here.

Reply:
This path can't be hit when `BUILDCACHE_IMPERSONATE` is set, and I've documented that where you suggested.